sources/models/shared-defaults/kubernetes-aws.toml

```diff
@@ -21,3 +21,11 @@ service-checks = ["apiserver", "chronyd", "containerd", "host-containerd", "kube
 
 [metadata.settings.network]
 affected-services = ["containerd", "kubernetes", "host-containerd", "host-containers"]
+
+# AutoScaling Warm Pool
+[settings.autoscaling]
+should-wait = false
```

Comment:
Can we make sure all `aws` variants have `AutoscalingSettings` in their model, and just move this to the common `aws` settings instead?